app/src/backend/mature.js

```diff
@@ -1,8 +1,7 @@
+import {matureWords} from "./matureWords.json"
```

Comment:
The netlify function is built using `esbuild` which as far as I understood doesn't allow directly importing of .json's like this. Maybe better to replace with a `readFile` operation

Reply:
makes sense, i'll update it